engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -2091,8 +2091,11 @@ public void execute(String[] args, ConsoleReader reader) throws Exception
           if (!commandLineInfo.localMode) {
 
             // see whether there is an app with the same name and user name running
-            String appNameAttributeName = StreamingApplication.DT_PREFIX + Context.DAGContext.APPLICATION_NAME.getName();
-            String appName = config.get(appNameAttributeName, appFactory.getName());
+            String appNameAttributeName = StreamingApplication.APEX_PREFIX + Context.DAGContext.APPLICATION_NAME.getName();
+            String appName = config.get(appNameAttributeName);
```

Comment:
I guess it will be there till next major release, but regardless: addDeprecation doesn't really help here, it is for situations where lookup occurs with old key and new key is used as replacement. Here I have already changed the code to use the new key.